lib/uksched/include/uk/thread.h

```diff
@@ -579,9 +862,39 @@ struct uk_thread *uk_thread_create_fn2(struct uk_alloc *a,
 				       void *priv,
 				       uk_thread_dtor_t dtor);
 
+/**
+ * Releases the resources of a given thread.
+ *
+ * @param t
+ *   Pointer to the thread
+ */
 void uk_thread_release(struct uk_thread *t);
+
+/**
+ * Blocks a thread for a given time.
+ *
+ * @param thread
+ *   Pointer to the thread
+ *
+ * @param nsec
+ *   Number of nanoseconds for the thread to be blocked.
```

Comment:
```suggestion
 * @param thread
 *   Pointer to the thread
 * @param nsec
 *   Number of nanoseconds for the thread to be blocked.
```
Leave empty lines only between the function brief and parameters/return descriptions.